docs/octopus-rest-api/examples/delete-machines-by-role.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Delete machines by role
+description: An example script to delete machines by role.
+---
+
+
+These scripts demonstrate how to programmatically delete machines with the specified role from the Octopus Server.
+
+:::warning
+These scripts will remove the machines from the space on the Octopus Server, use with caution
+:::
```

Comment:
I'm removing the warning as the users has already encounter two pieces of text that state this.
```suggestion
```